pkg/minikube/cluster/cluster.go

```diff
@@ -75,27 +76,27 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	glog.Infoln("Machine exists!")
 	h, err := api.Load(constants.MachineName)
 	if err != nil {
-		return nil, fmt.Errorf(
+		return nil, errors.Errorf(
```

Comment:
I was not sure how to change this as the err was embedded in the original message.  Should I remote the fmt and use Wrap, is this how it should read?
		return nil, errors.Errorf(
			"Error loading existing host: %s. Please try running [minikube delete], then run [minikube start] again.", err)

BECOMES
		return nil, errors.Wrap(err,
			"Error loading existing host. Please try running [minikube delete], then run [minikube start] again.")